scriptmodules/supplementary/skyscraper.sh

```diff
@@ -199,44 +203,32 @@ function configure_skyscraper() {
 function _init_config_skyscraper() {
     local scraper_conf_dir="$configdir/all/skyscraper"
 
-    # Make sure the `artwork.xml` and other conf file(s) are present, but don't overwrite them on upgrades
-    local f_conf
-    for f_conf in artwork.xml aliasMap.csv; do
-        if [[ -f "$scraper_conf_dir/$f_conf" ]]; then
-            cp -f "$md_inst/$f_conf" "$scraper_conf_dir/$f_conf.default"
-        else
-            cp "$md_inst/$f_conf" "$scraper_conf_dir"
-        fi
-    done
-
     # If we don't have a previous config.ini file, copy the example one
-    if [[ ! -f "$scraper_conf_dir/config.ini" ]]; then
-        cp "$md_inst/config.ini.example" "$scraper_conf_dir/config.ini"
-        sed -i 's/\[esgamelist\]/[esgamelist]\ncacheScreenshots="false"/' "$scraper_conf_dir/config.ini"
```

Comment:
This was lost somehow and I think should reverted. This prevents the caching of artwork when importing images from EmulationStation.

Reply:
This is now implemented upstream in the [config.ini.example file](https://github.com/Gemba/skyscraper/blob/e8969834a6f00fbd06b64112b7363e0606ad4c29/config.ini.example#L169) that is copied from the repo, so I think we no longer need to insert this ourselves: https://github.com/Gemba/skyscraper/blob/e8969834a6f00fbd06b64112b7363e0606ad4c29/config.ini.example#L169

Reply:
It's ok then.